disco_aws_automation/disco_ssm.py

```diff
@@ -130,17 +131,10 @@ def _print_ssm_output(self, output):
         for instance, instance_output in output.iteritems():
             print("Output for instance: {}".format(instance))
             for plugin in instance_output:
-                print(
-                    u"Plugin: {}\n\n"
-                    u"STDOUT:\n{}\n\n"
-                    u"STDERR:\n{}\n\n"
-                    u"Exit Code: {}".format(
-                        plugin.get('name', '-'),
-                        plugin.get('stdout', '-'),
-                        plugin.get('stderr', '-'),
-                        plugin.get('exit_code')
-                    )
-                )
+                print(u"Plugin: {}\n\n".format(plugin.get('name', '-')))
```

Comment:
Is there an actual significance to the multiple print statements? Otherwise it looks like the only change was setting a default for exit_code, correct?

Reply:
looks like the issue was that we were trying to pass to `print` several not converted to unicode strings